my-vue3-app/src/components/SubmitReport.vue

```diff
@@ -1,10 +1,25 @@
 <template>
-  <div class="hello">
+  <!-- <div class="hello">
     <h1>{{ msg }}</h1>
     <p>
       Сайт для наркетов!
     </p>
     <h3>Installed CLI Plugins</h3>
+  </div> -->
+  <div class="submitReportForm">
+    <h1 class="submitReportForm__title">Расскажите о ваших впечатлениях в подробностях</h1>
+      <input v-model="title" placeholder="Add title">
+      <p>The title is: {{ title }}</p>
```

Comment:
It is not necessary to see it. Let's remove it

Reply:
not necessary to see what?

Reply:
The whole line. When you enter field called 'title', you don't need to see under the field this <p>